.github/workflows/push-build.yaml

```diff
@@ -51,15 +51,14 @@ jobs:
           echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
           echo "GITHUB_REF"=${GITHUB_REF} >> $GITHUB_ENV
 
-      - name: install dependencies
+      - name: install dependencies and configure git user
         run: |
           python3 -m pip install --upgrade pip setuptools wheel
-          wget https://github.com/libgit2/libgit2/archive/refs/tags/v1.3.1.tar.gz -O libgit2-1.3.1.tar.gz
-          tar xzf libgit2-1.3.1.tar.gz
-          cd libgit2-1.3.1/
-          cmake .
-          make
-          sudo make install
+          sudo apt-get -q update
+          sudo apt-get -q install -y libgit2-dev
+          # Use our Rook 'service account'
+          git config user.name Rook
+          git config user.email "cncf-rook-info@lists.cncf.io"
```

Comment:
We don't have this email anymore. It's just for the commit messages, right? That's fine since anyway we must have been setting that before too.

Reply:
Yes, it should only be used for the commit message. That email is/was used in the rook/rook.github.io repo as well for commiting the docs changes :D